igvm/vm.py

```diff
@@ -197,6 +201,47 @@ def start(self, force_stop_failed=True, transaction=None):
         if transaction:
             transaction.on_rollback('stop VM', self.shutdown)
 
+    def aws_start(self):
```

Comment:
This function is never called. So you provided only low level api. Also is it aws_start or aws_build?

Reply:
Its called in `vm_build` in commands.py and it is starting an already build VM in AWS.